C-Plus-Plus/ds/ThreadedBinaryTree.cpp

```diff
@@ -0,0 +1,113 @@
+#include <iostream>
+#include<algorithm>
+#include<cstdlib>
+#include <math.h>
+#define COUNT 10
+using namespace std;
+
+typedef struct  Node
+{
+  int data;
+  int leftflag,rightflag; // 1 for address of child, 0 address of parent
```

Comment:
Use Comments above the Line of Code; Not beside it. Please make changes at other places as well.